### DISCUSSION TITLE
Updated to 3.9.9.9 and I get Insert Disk Error (Steam FF7)

### DISCUSSION THREAD

**Discussion by mattyflip** [tsunamods-codes/7th-Heaven#197] [Q&A] [answered]
All I did was update to 3.9.9.9 and when I went to launch the game it asked me to insert disk 1, 2 or 3
I've tried everything except reinstall 7th heaven on a previous version. 
I don't want to lose my saved game files.

** Versions **
[3.9.9.9] Tested on latest stable version
[3.9.9.9 ] Pre-release only bug

**Desktop (please complete the following information):**
 - OS: [Windows 11]
 - Version: [21H2]
 - Build: [22000.2538]

**Additional context**
I've reinstalled the game through steam, restarted my PC multiple times, created an exception in my firewall for 7th heaven...
I just don't know what to do any more and I cant find any solutions on any other forums 😢

**Comment by julianxhokaxhiu** [comment, marked as answer]
It's intentional, the canaries now are meant to be used either natively through the Steam edition or if you own the 1998 one, with the original disks, just insert it and 7th will autodetect it for you.

If you own the Steam game, uninstall 7th heaven and reinstall it. It will autodetect the Steam game edition ( make sure you've installed the English version ), hit Save on the general settings and play. You're now ready to play with any mod you like.

Save files can be either put again next to the path where your FF7 Exe is detected by 7th Heaven, or in the original Square Enix folder in documents, see https://savelocation.net/ff7

**Reply by mattyflip** [reply]
Thank you so much! That worked perfect! I was getting so frustrated 😅 I just didnt want to lose my save data and mods list haha

**Reply by julianxhokaxhiu** [reply]
No worries, you're welcome :) Enjoy!
